app.js

```diff
@@ -34,7 +36,20 @@ app.use(bodyParser.urlencoded({extended: false}));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/messages', routes);
+app.use('/messages', messagesRoute);
+
+app.use('/long/messages', longPollingMessagesRoute);
+
+//timeout catcher
+app.use(function (req, res, next) {
```

Comment:
who is going to first? route to /long/messages or timeout?
does the order in app.js is important?